book/website/community-handbook/local-build.md

```diff
@@ -47,33 +72,64 @@ You will need to locate your "terminal" or "prompt" application on your machine.
+
+
+### Check external links in the book
+
+When editing or reviewing this book locally, you can run the Sphinx link checker with Jupyter Book to check if the external links mentioned in the book are valid.
+To run the link checker, use the following command:
+
+```bash
+cd book/website
+jupyter-book build . --builder linkcheck
+```
+
+The link checker checks if the each link resolves and prints the status on your terminal so that you can check and resolve any incorrect links.
+Read more about this on the [Jupyter Book's GitHub repository](https://github.com/executablebooks/jupyter-book/blob/master/docs/advanced/advanced.md#check-external-links-in-your-book).
+
 ## Why did we recommend using (mini)conda?
-In the step-by-step guide above, we made use of the `jupyter-book` command to build the Turing Way book. For this command to work as intended you will need a Python installation on your machine. 
+
+In the step-by-step guide above, we made use of the `jupyter-book` command to build the Turing Way book. For this command to work as intended you will need a Python installation on your machine.
 As with any other programming language such as R or Julia, any given Python installation might look different from another due to the different packages or libraries that come with the installation.
 Over time you will likely install even more packages, or update packages to newer versions. Some packages also depend on the presence of specific versions of other packages to function, and so to ensure your local build works smoothly you will want to minimize as much mismatched dependencies as possible.
 
-But this can be difficult! Even with an organized, concerted effort, package management for programming languages naturally throws up dependency issues. Python packages, for reasons not discussed here, tend to suffer from dependency issues a bit more than other languages (note that all languages do!) and one guaranteed way to come across such an issue by trying to maintain all of your Python projects using just one, large set of packages, each at a specific version. You simply can't cater to the needs of all package dependencies this way. https://xkcd.com/1987/   
+But this can be difficult! Even with an organized, concerted effort, package management for programming languages naturally throws up dependency issues. Python packages, for reasons not discussed here, tend to suffer from dependency issues a bit more than other languages (note that all languages do!) and one guaranteed way to come across such an issue by trying to maintain all of your Python projects using just one, large set of packages, each at a specific version. You simply can't cater to the needs of all package dependencies this way.
+
 ![](https://imgs.xkcd.com/comics/python_environment.png)
```

Comment:
I think we are falling short of the CC BY-NC requirements here. We need to provide attribution and a link to the licence.

The image also has no caption or alt-text.

I think we could kill two birds with one stone here by adding a caption with a link to the xkcd page (which should be sufficient for attribution, showing changes and licence). We should also have alt-text explaining the image. Might be worth asking @the-turing-way/accessibility for advice.

Reply:
Absolutely! I forgot to mention that I did see this when reviewing the page, but thought that's a separate issue than the one I'm addressing in this PR, but if you think we should do it here anyway, I can try to do something to this effect!

Reply:
I think it would be good to try and fix it here as we are improving that page in this PR and it's a bit of a bad look to pass over attribution and accessibility problems.

We can open an issue instead if it looks to be a more major task. What do you think @kallewesterling?

Reply:
No, no! I'll just spend a few minutes on it now, to make sure it's rectified in this PR!

Reply:
Please have a look at https://github.com/the-turing-way/the-turing-way/pull/3446/commits/2e64e47f4672fe18c7da429002ad37e9fd654b48